README.adoc

```diff
@@ -161,15 +161,15 @@ persistence.xml
 include::finish/backendServices/src/main/resources/META-INF/persistence.xml[tags=**;]
 ----
 
-The persistence unit is defined by the [hotspot=7-13 file=0]`<persistence-unit/>` XML element. The [hotspot=7 file=0]`name` attribute is 
+The persistence unit is defined by the [persistence-unit file=0]`<persistence-unit/>` XML element. The [hotspot=transaction-type file=0]`name` attribute is 
```

Comment:
`<persistence-unit/>` is an invalid XML tag format, if referencing the tag, just remove the `/` to make `<persistence-unit>` otherwise use `</persistence-unit>` for an end tag reference

Reply:
Good catch. I would suggest opening an issue in `guides-common`. The hotspot '"invalid xml tag" for the hotspot reference is also included in other guides.

Reply:
https://github.com/OpenLiberty/guides-common/issues/355